src/clr_bundle_rm.c

```diff
@@ -191,84 +191,11 @@ static bool parse_options(int argc, char **argv)
 	return false;
 }
 
-static void reload_parsed_opts(void)
-{
-	if (curopts.path_prefix) {
```

Comment:
I think you can get rid of the curopts struct as you are not using the reload_parsed_opts() anymore. can you check that?

Reply:
good catch, I'll remove that in another commit.